mycpp/gc_builtins.cc

```diff
@@ -343,3 +353,15 @@ int max(List<int>* elems) {
 
   return ret;
 }
+
+Str* raw_input(Str* prompt) {
+#ifdef HAVE_READLINE
+  char* ret = readline(prompt->data());
+  if (ret == nullptr) {
+    throw Alloc<EOFError>();
+  }
+  return StrFromC(ret, strlen(ret));
+#else
+  assert(0);  // not implemented
```

Comment:
I think we can fill this in pretty easily to be something like `LineReader.readline()`, but we can do it in another PR

Maybe we need a global `CFileLineReader(mylib.Stdin())`  not sure?

Or there is also `ReadLine()` for the `read` builtin

It's complex because of buffering -- shells do some weird things there